<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> Stuart A. MacGowan </title> <meta name="author" content="Stuart A. MacGowan"> <meta name="description" content="A simple, whitespace theme for academics. Based on [*folio](https://github.com/bogoli/-folio) design. "> <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%E2%9A%9B%EF%B8%8F&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://stuartmac.github.io/"> <script src="/assets/js/theme.js?a5ca4084d3b81624bcfa01156dae2b8e"></script> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script>initTheme();</script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item active"> <a class="nav-link" href="/">about <span class="sr-only">(current)</span> </a> </li> <li class="nav-item "> <a class="nav-link" href="/publications/">publications </a> </li> <li class="nav-item "> <a class="nav-link" href="/repositories/">repositories </a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">cv </a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="ti ti-sun-moon" id="light-toggle-system"></i> <i class="ti ti-moon-filled" id="light-toggle-dark"></i> <i class="ti ti-sun-filled" id="light-toggle-light"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title"> <span class="font-weight-bold">Stuart</span> A. MacGowan </h1> <p class="desc"><a href="#">Computational Biologist</a>. University of Dundee. smacgowan@dundee.ac.uk</p> </header> <article> <div class="profile float-right"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/prof_pic-480.webp 480w,/assets/img/prof_pic-800.webp 800w,/assets/img/prof_pic-1400.webp 1400w," sizes="(min-width: 800px) 231.0px, (min-width: 576px) 30vw, 95vw" type="image/webp"> <img src="/assets/img/prof_pic.jpg?0ac4684dbdd2e2d9c8d2762b7082b9bb" class="img-fluid z-depth-1 rounded" width="100%" height="auto" alt="prof_pic.jpg" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </source></picture> </figure> <div class="more-info"> <p>School of Life Sciences</p> <p>University of Dundee</p> <p>Dundee, Scotland</p> </div> </div> <div class="clearfix"> <h2 id="biography">Biography</h2> <p>Dr. Stuart MacGowan is a Senior Bioinformatics Scientist in Prof. Geoff Barton’s group at the University of Dundee. Alongside his research into understanding the intricacies of protein sequences, structures, and human genetic variation, he also maintains and manages the Dundee Resource for Sequence Analysis and Structure Prediction (DRSASP).</p> <p>Dr. MacGowan has dedicated his career to applying computational and statistical techniques to address key questions in biology and chemistry. After completing his PhD in 2014 under the guidance of Prof. Mathias Senge at Trinity College Dublin, he joined the Data Analysis Group (DAG) within Geoff Barton’s group at the University of Dundee. After a few years working on collaborative projects with the DAG, including multiple projects with Prof. Irwin McLean, he shifted his attention to computational work on genetic variation in proteins.</p> <p>In 2018, Dr. MacGowan was promoted to Senior Bioinformatics Scientist and appointed as the Manager of the Dundee Resource for Sequence Analysis and Structure Prediction. In this role, he oversees the development, maintenance, and enhancement of DRSASP, ensuring that it remains a valuable resource for the scientific community.</p> <h2 id="research">Research</h2> <p>At Dundee, Dr. MacGowan’s research has involved collaborations with multidisciplinary teams of scientists and clinicians to identify genetic variants and candidate genes implicated in rare genetic skin disorders, atopic dermatitis and COVID-19, alongside his own theoretical work on how human population variation relates to evolutionary changes in proteins.</p> <p>A cornerstone of his research is understanding the influence of genetic variation on protein structure and function, with a special focus on how variations in ligand and protein binding sites impact the overall function of the protein. He has approached this problem from the “big data” perspective, analysing sequence and structure data from hundreds of thousands of proteins to identify universal trends and patterns. He has also applied these approaches to more specific situations, such as investigating the role of common and rare mutations in the human ACE2 gene and their effect on COVID-19.</p> <p>In recent years, Dr. MacGowan has maintained DRSASP, which aims to provide a comprehensive toolkit for analysing protein sequences and structures, focusing on aspects such as protein structure, disorder, crystallization, and protein-protein interactions. In addition to maintaining and enhancing this resource, Dr. MacGowan is currently working to expand its capabilities to encompass genetic variation, improve the resources reliability, and ensure the reproducibility of analyses for the future.</p> </div> <h2> <a href="/news/" style="color: inherit">news</a> </h2> <div class="news"> <div class="table-responsive"> <table class="table table-sm table-borderless"> <tr> <th scope="row" style="width: 20%">Mar 28, 2024</th> <td> My latest article was just formally accepted accepted by <a href="https://www.nature.com/commsbio/" rel="external nofollow noopener" target="_blank">Communicatons Biology</a>! <img class="emoji" title=":tada:" alt=":tada:" src="https://github.githubassets.com/images/icons/emoji/unicode/1f389.png" height="20" width="20"> I’m excited to share the article with you once it’s published. Stay tuned for more updates! <img class="emoji" title=":sparkles:" alt=":sparkles:" src="https://github.githubassets.com/images/icons/emoji/unicode/2728.png" height="20" width="20"> <img class="emoji" title=":smile:" alt=":smile:" src="https://github.githubassets.com/images/icons/emoji/unicode/1f604.png" height="20" width="20"> </td> </tr> <tr> <th scope="row" style="width: 20%">Mar 27, 2024</th> <td> Lauched <a href="#">my personal website here on GitHub Pages</a>! I’m excited to share my research, publications, and other projects with the world. I hope you find the information here useful and informative. Feel free to reach out if you have any questions or comments. Thanks for visiting! <img class="emoji" title=":sparkles:" alt=":sparkles:" src="https://github.githubassets.com/images/icons/emoji/unicode/2728.png" height="20" width="20"> <img class="emoji" title=":smile:" alt=":smile:" src="https://github.githubassets.com/images/icons/emoji/unicode/1f604.png" height="20" width="20"> </td> </tr> </table> </div> </div> <h2> <a href="/blog/" style="color: inherit">latest posts</a> </h2> <div class="news"> <div class="table-responsive"> <table class="table table-sm table-borderless"> </table> </div> </div> <h2> <a href="/publications/" style="color: inherit">selected publications</a> </h2> <div class="publications"> <ol class="bibliography"> <li> <div class="row"> <div id="Utges2024-pn" class="col-sm-10"> <div class="title">Classification of likely functional class for ligand binding sites identified from fragment screening</div> <div class="author"> Javier S Utgés , Stuart A MacGowan , Callum M Ives , and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Geoffrey J Barton' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>Commun Biol</em>, Mar 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="http://dx.doi.org/10.1038/s42003-024-05970-8" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">URL</a> <a href="https://doi.org/10.1038/s42003-024-05970-8" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Fragment screening is used to identify binding sites and leads in drug discovery, but it is often unclear which binding sites are functionally important. Here, data from 37 experiments, and 1309 protein structures binding to 1601 ligands were analysed. A method to group ligands by binding sites is introduced and sites clustered according to profiles of relative solvent accessibility. This identified 293 unique ligand binding sites, grouped into four clusters (C1-4). C1 includes larger, buried, conserved, and population missense-depleted sites, enriched in known functional sites. C4 comprises smaller, accessible, divergent, missense-enriched sites, depleted in functional sites. A site in C1 is 28 times more likely to be functional than one in C4. Seventeen sites, which to the best of our knowledge are novel, in 13 proteins are identified as likely to be functionally important with examples from human tenascin and 5-aminolevulinate synthase highlighted. A multi-layer perceptron, and K-nearest neighbours model are presented to predict cluster labels for ligand binding sites with an accuracy of 96% and 100%, respectively, so allowing functional classification of sites for proteins not in this set. Our findings will be of interest to those studying protein-ligand interactions and developing new drugs or function modulators.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="MacGowan2023-rf" class="col-sm-10"> <div class="title">A unified approach to evolutionary conservation and population constraint in protein domains highlights structural features and pathogenic sites</div> <div class="author"> Stuart A MacGowan , <a href="https://www.ebi.ac.uk/people/person/fabio-madeira/" rel="external nofollow noopener" target="_blank">Fábio Madeira</a>, <a href="https://github.com/tbrittoborges/tbrittoborges" rel="external nofollow noopener" target="_blank">Thiago Britto-Borges</a>, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Geoffrey J Barton' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> Jul 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.researchsquare.com/article/rs-3160340/v1" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">URL</a> <a href="https://doi.org/10.21203/rs.3.rs-3160340/v1" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Abstract The molecular evolution of a protein is constrained by its structure and function. This is how patterns of evolutionary conservation in a multiple sequence alignment can be exploited by algorithms like AlphaFold to predict structure and other features. Human population sequencing offers the potential to show similar trends within a single species. However, although gene-level and domain-level aggregation of variants is routinely applied, data still preclude simple residue-level analysis per protein. Here, we aggregate population variants within protein domain families to define the missense enrichment score (MES) which captures population constraint at positions in the family. Contrasting MES with evolutionary conservation identifies positions that are key to protein-ligand and protein-protein interaction specificity as well as locations critical to folding, function and pathogenicity. This approach also highlights a new class of pathogenic variant hot spot at positions that are evolutionarily conserved yet enriched in missense variants. We found 5,086 positions in 766 domain families were missense depleted (covering 365,300 residues in the human proteome) and 13,490 positions in 3,591 families were enriched in missense variants (340,829 residues in human) and provide a map of residue-level population constraint in human. Comparison to 61,214 three dimensional structures from the Protein Data Bank, covering 40,394 sequences from 3,661 Pfam domains, and over 10,000 variants labelled pathogenic in ClinVar provides validation of these sites in context with structure and pathogenicity. Application to the GPCR family identifies 17 sites important for ligand binding specificity as well as sites key to function in all members of the GPCR family. For nuclear receptor ligand binding domains, of 10 likely specificity sites, six are in direct contact with ligands. In P450s the most conserved and missense depleted sites interact with the electron donor. This study will have broad applications to: (a) providing focus for functional studies of specific proteins by mutagenesis; (b) refining pathogenicity prediction models; (c) highlighting which residue interactions to target when refining the specificity of small-molecule drugs.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Cleghorn2023-gb" class="col-sm-10"> <div class="title">Development of a 2,4-Diaminothiazole Series for the Treatment of Human African Trypanosomiasis Highlights the Importance of Static-Cidal Screening of Analogues</div> <div class="author"> Laura A T Cleghorn , Richard J Wall , Sébastien Albrecht , and <span class="more-authors" title="click to view 19 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '19 more authors' ? 'Stuart A MacGowan, Suzanne Norval, Manu De Rycker, Andrew Woodland, Daniel Spinks, Stephen Thompson, Stephen Patterson, Victoriano Corpas Lopez, Gourav Dey, Iain T Collie, Irene Hallyburton, Robert Kime, Frederick R C Simeons, Laste Stojanovski, Julie A Frearson, Paul G Wyatt, Kevin D Read, Ian H Gilbert, Susan Wyllie' : '19 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">19 more authors</span> </div> <div class="periodical"> <em>J. Med. Chem.</em>, Jul 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="http://dx.doi.org/10.1021/acs.jmedchem.3c00509" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">URL</a> <a href="https://doi.org/10.1021/acs.jmedchem.3c00509" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>While treatment options for human African trypanosomiasis (HAT) have improved significantly, there is still a need for new drugs with eradication now a realistic possibility. Here, we report the development of 2,4-diaminothiazoles that demonstrate significant potency against Trypanosoma brucei, the causative agent of HAT. Using phenotypic screening to guide structure-activity relationships, potent drug-like inhibitors were developed. Proof of concept was established in an animal model of the hemolymphatic stage of HAT. To treat the meningoencephalitic stage of infection, compounds were optimized for pharmacokinetic properties, including blood-brain barrier penetration. However, in vivo efficacy was not achieved, in part due to compounds evolving from a cytocidal to a cytostatic mechanism of action. Subsequent studies identified a nonessential kinase involved in the inositol biosynthesis pathway as the molecular target of these cytostatic compounds. These studies highlight the need for cytocidal drugs for the treatment of HAT and the importance of static-cidal screening of analogues.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="MacGowan2022-jy" class="col-sm-10"> <div class="title">Missense variants in human ACE2 strongly affect binding to SARS-CoV-2 Spike providing a mechanism for ACE2 mediated genetic risk in Covid-19: A case study in affinity predictions of interface variants</div> <div class="author"> Stuart A MacGowan , Michael I Barton , Mikhail Kutuzov , and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Omer Dushek, P Anton Merwe, Geoffrey J Barton' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>PLoS Comput. Biol.</em>, Mar 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="http://dx.doi.org/10.1371/journal.pcbi.1009922" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">URL</a> <a href="https://doi.org/10.1371/journal.pcbi.1009922" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>SARS-CoV-2 Spike (Spike) binds to human angiotensin-converting enzyme 2 (ACE2) and the strength of this interaction could influence parameters relating to virulence. To explore whether population variants in ACE2 influence Spike binding and hence infection, we selected 10 ACE2 variants based on affinity predictions and prevalence in gnomAD and measured their affinities and kinetics for Spike receptor binding domain through surface plasmon resonance (SPR) at 37°C. We discovered variants that reduce and enhance binding, including three ACE2 variants that strongly inhibited (p.Glu37Lys, ∆\DeltaG = -1.33 \pm 0.15 kcal mol-1 and p.Gly352Val, predicted ∆\DeltaG = -1.17 kcal mol-1) or abolished (p.Asp355Asn) binding. We also identified two variants with distinct population distributions that enhanced affinity for Spike. ACE2 p.Ser19Pro (∆\DeltaG = 0.59 \pm 0.08 kcal mol-1) is predominant in the gnomAD African cohort (AF = 0.003) whilst p.Lys26Arg (∆\DeltaG = 0.26 \pm 0.09 kcal mol-1) is predominant in the Ashkenazi Jewish (AF = 0.01) and European non-Finnish (AF = 0.006) cohorts. We compared ACE2 variant affinities to published SARS-CoV-2 pseudotype infectivity data and confirmed that ACE2 variants with reduced affinity for Spike can protect cells from infection. The effect of variants with enhanced Spike affinity remains unclear, but we propose a mechanism whereby these alleles could cause greater viral spreading across tissues and cell types, as is consistent with emerging understanding regarding the interplay between receptor affinity and cell-surface abundance. Finally, we compared mCSM-PPI2 ∆\DeltaG predictions against our SPR data to assess the utility of predictions in this system. We found that predictions of decreased binding were well-correlated with experiment and could be improved by calibration, but disappointingly, predictions of highly enhanced binding were unreliable. Recalibrated predictions for all possible ACE2 missense variants at the Spike interface were calculated and used to estimate the overall burden of ACE2 variants on Covid-19.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="MacGowan2020-bd" class="col-sm-10"> <div class="title">The Dundee Resource for Sequence Analysis and Structure Prediction</div> <div class="author"> Stuart A MacGowan , <a href="https://www.ebi.ac.uk/people/person/fabio-madeira/" rel="external nofollow noopener" target="_blank">Fábio Madeira</a>, <a href="https://github.com/tbrittoborges/tbrittoborges" rel="external nofollow noopener" target="_blank">Thiago Britto-Borges</a>, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Mateusz Warowny, Alexey Drozdetskiy, James B Procter, Geoffrey J Barton' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em>Protein Sci.</em>, Jan 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="http://dx.doi.org/10.1002/pro.3783" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">URL</a> <a href="https://doi.org/10.1002/pro.3783" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>The Dundee Resource for Sequence Analysis and Structure Prediction (DRSASP; http://www.compbio.dundee.ac.uk/drsasp.html) is a collection of web services provided by the Barton Group at the University of Dundee. DRSASP’s flagship services are the JPred4 webserver for secondary structure and solvent accessibility prediction and the JABAWS 2.2 webserver for multiple sequence alignment, disorder prediction, amino acid conservation calculations, and specificity-determining site prediction. DRSASP resources are available through conventional web interfaces and APIs but are also integrated into the Jalview sequence analysis workbench, which enables the composition of multitool interactive workflows. Other existing Barton Group tools are being brought under the banner of DRSASP, including NoD (Nucleolar localization sequence detector) and 14-3-3-Pred. New resources are being developed that enable the analysis of population genetic data in evolutionary and 3D structural contexts. Existing resources are actively developed to exploit new technologies and maintain parity with evolving web standards. DRSASP provides substantial computational resources for public use, and since 2016 DRSASP services have completed over 1.5 million jobs.</p> </div> </div> </div> </li> </ol> </div> <div class="social"> <div class="contact-icons"> <a href="mailto:%73%6D%61%63%67%6F%77%61%6E@%64%75%6E%64%65%65.%61%63.%75%6B" title="email"><i class="fa-solid fa-envelope"></i></a> <a href="https://orcid.org/0000-0003-4233-5071" title="ORCID" rel="external nofollow noopener" target="_blank"><i class="ai ai-orcid"></i></a> <a href="https://scholar.google.com/citations?user=q85awyIAAAAJ" title="Google Scholar" rel="external nofollow noopener" target="_blank"><i class="ai ai-google-scholar"></i></a> <a href="https://github.com/stuartmac" title="GitHub" rel="external nofollow noopener" target="_blank"><i class="fa-brands fa-github"></i></a> <a href="https://www.linkedin.com/in/stuart-macgowan" title="LinkedIn" rel="external nofollow noopener" target="_blank"><i class="fa-brands fa-linkedin"></i></a> <a href="https://twitter.com/stuartmac44" title="X" rel="external nofollow noopener" target="_blank"><i class="fa-brands fa-x-twitter"></i></a> <a href="/feed.xml" title="RSS Feed"><i class="fa-solid fa-square-rss"></i></a> </div> <div class="contact-note">The best way to contact me is by <a href="https://www.linkedin.com/in/stuart-macgowan" rel="external nofollow noopener" target="_blank">DM on LinkedIn</a>. You can also <a href="https://twitter.com/stuartmac44" rel="external nofollow noopener" target="_blank">follow me on X</a> or if you prefer you can <a href="mailto:smacgowan@dundee.ac.uk">send me an email</a>. </div> </div> </article> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> © Copyright 2024 Stuart A. MacGowan. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. Photos from <a href="https://unsplash.com" target="_blank" rel="external nofollow noopener">Unsplash</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?5d75c11f89cd96294bf5e6dd1ee1bb30"></script> <script defer src="/assets/js/common.js?fcfacfb8c6281f5e68d5a7d348186eb1"></script> <script defer src="/assets/js/copy_code.js?12775fdf7f95e901d7119054556e495f" type="text/javascript"></script> <script defer src="/assets/js/jupyter_new_tab.js?d9f17b6adc2311cbabd747f4538bb15f"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6"></script> <script async src="https://www.googletagmanager.com/gtag/js?id=G-ZNT0W2MVX3"></script> <script>function gtag(){window.dataLayer.push(arguments)}window.dataLayer=window.dataLayer||[],gtag("js",new Date),gtag("config","G-ZNT0W2MVX3");</script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> </body> </html>