<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> publications | Stuart A. MacGowan </title> <meta name="author" content="Stuart A. MacGowan"> <meta name="description" content="Stuart MacGowan's list of publications."> <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%E2%9A%9B%EF%B8%8F&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://stuartmac.github.io/publications/"> <script src="/assets/js/theme.js?a5ca4084d3b81624bcfa01156dae2b8e"></script> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script>initTheme();</script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"> <span class="font-weight-bold">Stuart</span> A. MacGowan </a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about </a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">publications <span class="sr-only">(current)</span> </a> </li> <li class="nav-item "> <a class="nav-link" href="/repositories/">repositories </a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">cv </a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="ti ti-sun-moon" id="light-toggle-system"></i> <i class="ti ti-moon-filled" id="light-toggle-dark"></i> <i class="ti ti-sun-filled" id="light-toggle-light"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title">publications</h1> <p class="post-description">Stuart MacGowan's list of publications.</p> </header> <article> <div class="publications"> <h2 class="bibliography">2024</h2> <ol class="bibliography"> <li> <div class="row"> <div id="Wall2024-xr" class="col-sm-10"> <div class="title">ResMAP-a saturation mutagenesis platform enabling parallel profiling of target-specific resistance-conferring mutations in Plasmodium</div> <div class="author"> Richard J Wall , Stuart A MacGowan , Irene Hallyburton , and <span class="more-authors" title="click to view 8 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '8 more authors' ? 'Aisha J Syed, Sowmya Ajay Castro, Gourav Dey, Rachel Milne, Stephen Patterson, Jody Phelan, Natalie Wiedemar, Susan Wyllie' : '8 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">8 more authors</span> </div> <div class="periodical"> <em>MBio</em>, Oct 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="http://dx.doi.org/10.1128/mbio.01708-24" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">URL</a> <a href="https://doi.org/10.1128/mbio.01708-24" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>New and improved drugs are required for the treatment and ultimate eradication of malaria. The efficacy of front-line therapies is now threatened by emerging drug resistance; thus, new tools to support the development of drugs with a lower propensity for resistance are needed. Here, we describe the development of a RESistance Mapping And Profiling (ResMAP) platform for the identification of resistance-conferring mutations in Plasmodium drug targets. Proof-of-concept studies focused on interrogating the antimalarial drug target, Plasmodium falciparum lysyl tRNA synthetase (PfKRS). Saturation mutagenesis was used to construct a plasmid library encoding all conceivable mutations within a 20-residue span at the base of the PfKRS ATP-binding pocket. The superior transfection efficiency of Plasmodium knowlesi was exploited to generate a high coverage parasite library expressing PfKRS bearing all possible amino acid changes within this region of the enzyme. The selection of the library with PfKRS inhibitors, cladosporin and DDD01510706, successfully identified multiple resistance-conferring substitutions. Genetic validation of a subset of these mutations confirmed their direct role in resistance, with computational modeling used to dissect the structural basis of resistance. The application of ResMAP to inform the development of resistance-resilient antimalarials of the future is discussed. IMPORTANCE: An increase in treatment failures for malaria highlights an urgent need for new tools to understand and minimize the spread of drug resistance. We describe the development of a RESistance Mapping And Profiling (ResMAP) platform for the identification of resistance-conferring mutations in Plasmodium spp, the causative agent of malaria. Saturation mutagenesis was used to generate a mutation library containing all conceivable mutations for a region of the antimalarial-binding site of a promising drug target, Plasmodium falciparum lysyl tRNA synthetase (PfKRS). Screening of this high-coverage library with characterized PfKRS inhibitors revealed multiple resistance-conferring substitutions including several clinically relevant mutations. Genetic validation of these mutations confirmed resistance of up to 100-fold and computational modeling dissected their role in drug resistance. We discuss potential applications of this data including the potential to design compounds that can bypass the most serious resistance mutations and future resistance surveillance.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="MacGowan2024-no" class="col-sm-10"> <div class="title">A unified analysis of evolutionary and population constraint in protein domains highlights structural features and pathogenic sites</div> <div class="author"> Stuart A MacGowan , <a href="https://www.ebi.ac.uk/people/person/fabio-madeira/" rel="external nofollow noopener" target="_blank">Fábio Madeira</a>, <a href="https://github.com/tbrittoborges/tbrittoborges" rel="external nofollow noopener" target="_blank">Thiago Britto-Borges</a>, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Geoffrey J Barton' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>Commun. Biol.</em>, Apr 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.nature.com/articles/s42003-024-06117-5" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">URL</a> <a href="https://doi.org/10.1038/s42003-024-06117-5" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Protein evolution is constrained by structure and function, creating patterns in residue conservation that are routinely exploited to predict structure and other features. Similar constraints should affect variation across individuals, but it is only with the growth of human population sequencing that this has been tested at scale. Now, human population constraint has established applications in pathogenicity prediction, but it has not yet been explored for structural inference. Here, we map 2.4 million population variants to 5885 protein families and quantify residue-level constraint with a new Missense Enrichment Score (MES). Analysis of 61,214 structures from the PDB spanning 3661 families shows that missense depleted sites are enriched in buried residues or those involved in small-molecule or protein binding. MES is complementary to evolutionary conservation and a combined analysis allows a new classification of residues according to a conservation plane. This approach finds functional residues that are evolutionarily diverse, which can be related to specificity, as well as family-wide conserved sites that are critical for folding or function. We also find a possible contrast between lethal and non-lethal pathogenic sites, and a surprising clinical variant hot spot at a subset of missense enriched positions.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Utges2024-pn" class="col-sm-10"> <div class="title">Classification of likely functional class for ligand binding sites identified from fragment screening</div> <div class="author"> Javier S Utgés , Stuart A MacGowan , Callum M Ives , and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Geoffrey J Barton' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>Commun Biol</em>, Mar 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="http://dx.doi.org/10.1038/s42003-024-05970-8" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">URL</a> <a href="https://doi.org/10.1038/s42003-024-05970-8" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Fragment screening is used to identify binding sites and leads in drug discovery, but it is often unclear which binding sites are functionally important. Here, data from 37 experiments, and 1309 protein structures binding to 1601 ligands were analysed. A method to group ligands by binding sites is introduced and sites clustered according to profiles of relative solvent accessibility. This identified 293 unique ligand binding sites, grouped into four clusters (C1-4). C1 includes larger, buried, conserved, and population missense-depleted sites, enriched in known functional sites. C4 comprises smaller, accessible, divergent, missense-enriched sites, depleted in functional sites. A site in C1 is 28 times more likely to be functional than one in C4. Seventeen sites, which to the best of our knowledge are novel, in 13 proteins are identified as likely to be functionally important with examples from human tenascin and 5-aminolevulinate synthase highlighted. A multi-layer perceptron, and K-nearest neighbours model are presented to predict cluster labels for ligand binding sites with an accuracy of 96% and 100%, respectively, so allowing functional classification of sites for proteins not in this set. Our findings will be of interest to those studying protein-ligand interactions and developing new drugs or function modulators.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2023</h2> <ol class="bibliography"> <li> <div class="row"> <div id="MacGowan2023-rf" class="col-sm-10"> <div class="title">A unified approach to evolutionary conservation and population constraint in protein domains highlights structural features and pathogenic sites</div> <div class="author"> Stuart A MacGowan , <a href="https://www.ebi.ac.uk/people/person/fabio-madeira/" rel="external nofollow noopener" target="_blank">Fábio Madeira</a>, <a href="https://github.com/tbrittoborges/tbrittoborges" rel="external nofollow noopener" target="_blank">Thiago Britto-Borges</a>, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Geoffrey J Barton' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> Jul 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.researchsquare.com/article/rs-3160340/v1" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">URL</a> <a href="https://doi.org/10.21203/rs.3.rs-3160340/v1" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Abstract The molecular evolution of a protein is constrained by its structure and function. This is how patterns of evolutionary conservation in a multiple sequence alignment can be exploited by algorithms like AlphaFold to predict structure and other features. Human population sequencing offers the potential to show similar trends within a single species. However, although gene-level and domain-level aggregation of variants is routinely applied, data still preclude simple residue-level analysis per protein. Here, we aggregate population variants within protein domain families to define the missense enrichment score (MES) which captures population constraint at positions in the family. Contrasting MES with evolutionary conservation identifies positions that are key to protein-ligand and protein-protein interaction specificity as well as locations critical to folding, function and pathogenicity. This approach also highlights a new class of pathogenic variant hot spot at positions that are evolutionarily conserved yet enriched in missense variants. We found 5,086 positions in 766 domain families were missense depleted (covering 365,300 residues in the human proteome) and 13,490 positions in 3,591 families were enriched in missense variants (340,829 residues in human) and provide a map of residue-level population constraint in human. Comparison to 61,214 three dimensional structures from the Protein Data Bank, covering 40,394 sequences from 3,661 Pfam domains, and over 10,000 variants labelled pathogenic in ClinVar provides validation of these sites in context with structure and pathogenicity. Application to the GPCR family identifies 17 sites important for ligand binding specificity as well as sites key to function in all members of the GPCR family. For nuclear receptor ligand binding domains, of 10 likely specificity sites, six are in direct contact with ligands. In P450s the most conserved and missense depleted sites interact with the electron donor. This study will have broad applications to: (a) providing focus for functional studies of specific proteins by mutagenesis; (b) refining pathogenicity prediction models; (c) highlighting which residue interactions to target when refining the specificity of small-molecule drugs.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Cleghorn2023-gb" class="col-sm-10"> <div class="title">Development of a 2,4-Diaminothiazole Series for the Treatment of Human African Trypanosomiasis Highlights the Importance of Static-Cidal Screening of Analogues</div> <div class="author"> Laura A T Cleghorn , Richard J Wall , Sébastien Albrecht , and <span class="more-authors" title="click to view 19 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '19 more authors' ? 'Stuart A MacGowan, Suzanne Norval, Manu De Rycker, Andrew Woodland, Daniel Spinks, Stephen Thompson, Stephen Patterson, Victoriano Corpas Lopez, Gourav Dey, Iain T Collie, Irene Hallyburton, Robert Kime, Frederick R C Simeons, Laste Stojanovski, Julie A Frearson, Paul G Wyatt, Kevin D Read, Ian H Gilbert, Susan Wyllie' : '19 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">19 more authors</span> </div> <div class="periodical"> <em>J. Med. Chem.</em>, Jul 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="http://dx.doi.org/10.1021/acs.jmedchem.3c00509" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">URL</a> <a href="https://doi.org/10.1021/acs.jmedchem.3c00509" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>While treatment options for human African trypanosomiasis (HAT) have improved significantly, there is still a need for new drugs with eradication now a realistic possibility. Here, we report the development of 2,4-diaminothiazoles that demonstrate significant potency against Trypanosoma brucei, the causative agent of HAT. Using phenotypic screening to guide structure-activity relationships, potent drug-like inhibitors were developed. Proof of concept was established in an animal model of the hemolymphatic stage of HAT. To treat the meningoencephalitic stage of infection, compounds were optimized for pharmacokinetic properties, including blood-brain barrier penetration. However, in vivo efficacy was not achieved, in part due to compounds evolving from a cytocidal to a cytostatic mechanism of action. Subsequent studies identified a nonessential kinase involved in the inositol biosynthesis pathway as the molecular target of these cytostatic compounds. These studies highlight the need for cytocidal drugs for the treatment of HAT and the importance of static-cidal screening of analogues.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2022</h2> <ol class="bibliography"><li> <div class="row"> <div id="MacGowan2022-jy" class="col-sm-10"> <div class="title">Missense variants in human ACE2 strongly affect binding to SARS-CoV-2 Spike providing a mechanism for ACE2 mediated genetic risk in Covid-19: A case study in affinity predictions of interface variants</div> <div class="author"> Stuart A MacGowan , Michael I Barton , Mikhail Kutuzov , and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Omer Dushek, P Anton Merwe, Geoffrey J Barton' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>PLoS Comput. Biol.</em>, Mar 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="http://dx.doi.org/10.1371/journal.pcbi.1009922" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">URL</a> <a href="https://doi.org/10.1371/journal.pcbi.1009922" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>SARS-CoV-2 Spike (Spike) binds to human angiotensin-converting enzyme 2 (ACE2) and the strength of this interaction could influence parameters relating to virulence. To explore whether population variants in ACE2 influence Spike binding and hence infection, we selected 10 ACE2 variants based on affinity predictions and prevalence in gnomAD and measured their affinities and kinetics for Spike receptor binding domain through surface plasmon resonance (SPR) at 37°C. We discovered variants that reduce and enhance binding, including three ACE2 variants that strongly inhibited (p.Glu37Lys, ∆\DeltaG = -1.33 \pm 0.15 kcal mol-1 and p.Gly352Val, predicted ∆\DeltaG = -1.17 kcal mol-1) or abolished (p.Asp355Asn) binding. We also identified two variants with distinct population distributions that enhanced affinity for Spike. ACE2 p.Ser19Pro (∆\DeltaG = 0.59 \pm 0.08 kcal mol-1) is predominant in the gnomAD African cohort (AF = 0.003) whilst p.Lys26Arg (∆\DeltaG = 0.26 \pm 0.09 kcal mol-1) is predominant in the Ashkenazi Jewish (AF = 0.01) and European non-Finnish (AF = 0.006) cohorts. We compared ACE2 variant affinities to published SARS-CoV-2 pseudotype infectivity data and confirmed that ACE2 variants with reduced affinity for Spike can protect cells from infection. The effect of variants with enhanced Spike affinity remains unclear, but we propose a mechanism whereby these alleles could cause greater viral spreading across tissues and cell types, as is consistent with emerging understanding regarding the interplay between receptor affinity and cell-surface abundance. Finally, we compared mCSM-PPI2 ∆\DeltaG predictions against our SPR data to assess the utility of predictions in this system. We found that predictions of decreased binding were well-correlated with experiment and could be improved by calibration, but disappointingly, predictions of highly enhanced binding were unreliable. Recalibrated predictions for all possible ACE2 missense variants at the Spike interface were calculated and used to estimate the overall burden of ACE2 variants on Covid-19.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2021</h2> <ol class="bibliography"> <li> <div class="row"> <div id="Utges2021-ls" class="col-sm-10"> <div class="title">Ankyrin repeats in context with human population variation</div> <div class="author"> Javier S Utgés , Maxim I Tsenkov , Noah J M Dietrich , and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Stuart A MacGowan, Geoffrey J Barton' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>PLoS Comput. Biol.</em>, Aug 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="http://dx.doi.org/10.1371/journal.pcbi.1009335" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">URL</a> <a href="https://doi.org/10.1371/journal.pcbi.1009335" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Ankyrin protein repeats bind to a wide range of substrates and are one of the most common protein motifs in nature. Here, we collate a high-quality alignment of 7,407 ankyrin repeats and examine for the first time, the distribution of human population variants from large-scale sequencing of healthy individuals across this family. Population variants are not randomly distributed across the genome but are constrained by gene essentiality and function. Accordingly, we interpret the population variants in context with evolutionary constraint and structural features including secondary structure, accessibility and protein-protein interactions across 383 three-dimensional structures of ankyrin repeats. We find five positions that are highly conserved across homologues and also depleted in missense variants within the human population. These positions are significantly enriched in intra-domain contacts and so likely to be key for repeat packing. In contrast, a group of evolutionarily divergent positions are found to be depleted in missense variants in human and significantly enriched in protein-protein interactions. Our analysis also suggests the domain has three, not two surfaces, each with different patterns of enrichment in protein-substrate interactions and missense variants. Our findings will be of interest to those studying or engineering ankyrin-repeat containing proteins as well as those interpreting the significance of disease variants.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Barton2021-vg" class="col-sm-10"> <div class="title">Effects of common mutations in the SARS-CoV-2 Spike RBD and its ligand, the human ACE2 receptor on binding affinity and kinetics</div> <div class="author"> Michael I Barton , Stuart A MacGowan , Mikhail A Kutuzov , and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Omer Dushek, Geoffrey John Barton, P Anton Merwe' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>Elife</em>, Aug 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="http://dx.doi.org/10.7554/eLife.70658" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">URL</a> <a href="https://doi.org/10.7554/eLife.70658" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>The interaction between the SARS-CoV-2 virus Spike protein receptor binding domain (RBD) and the ACE2 cell surface protein is required for viral infection of cells. Mutations in the RBD are present in SARS-CoV-2 variants of concern that have emerged independently worldwide. For example, the B.1.1.7 lineage has a mutation (N501Y) in its Spike RBD that enhances binding to ACE2. There are also ACE2 alleles in humans with mutations in the RBD binding site. Here we perform a detailed affinity and kinetics analysis of the effect of five common RBD mutations (K417N, K417T, N501Y, E484K, and S477N) and two common ACE2 mutations (S19P and K26R) on the RBD/ACE2 interaction. We analysed the effects of individual RBD mutations and combinations found in new SARS-CoV-2 Alpha (B.1.1.7), Beta (B.1.351), and Gamma (P1) variants. Most of these mutations increased the affinity of the RBD/ACE2 interaction. The exceptions were mutations K417N/T, which decreased the affinity. Taken together with other studies, our results suggest that the N501Y and S477N mutations enhance transmission primarily by enhancing binding, the K417N/T mutations facilitate immune escape, and the E484K mutation enhances binding and immune escape.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2020</h2> <ol class="bibliography"> <li> <div class="row"> <div id="MacGowan2020-ay" class="col-sm-10"> <div class="title">Missense variants in ACE2 are predicted to encourage and inhibit interaction with SARS-CoV-2 Spike and contribute to genetic risk in COVID-19</div> <div class="author"> Stuart A MacGowan , and Geoffrey J Barton </div> <div class="periodical"> May 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.biorxiv.org/content/10.1101/2020.05.03.074781" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">URL</a> <a href="https://doi.org/10.1101/2020.05.03.074781" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Abstract SARS-CoV-2 invades host cells via an endocytic pathway that begins with the interaction of the SARS-CoV-2 Spike glycoprotein (S-protein) and human Angiotensin-converting enzyme 2 (ACE2). Genetic variability in ACE2 may be one factor that mediates the broad-spectrum severity of SARS-CoV-2 infection and COVID-19 outcomes. We investigated the capacity of ACE2 variation to influence SARS-CoV-2 infection with a focus on predicting the effect of missense variants on the ACE2 SARS-CoV-2 S-protein interaction. We validated the mCSM-PPI2 variant effect prediction algorithm with 26 published ACE2 mutant SARS-CoV S-protein binding assays and found it performed well in this closely related system (True Positive Rate = 0.7, True Negative Rate = 1). Application of mCSM-PPI2 to ACE2 missense variants from the Genome Aggregation Consortium Database (gnomAD) identified three that are predicted to strongly inhibit or abolish the S-protein ACE2 interaction altogether (p.Glu37Lys, p.Gly352Val and p.Asp355Asn) and one that is predicted to promote the interaction (p.Gly326Glu). The S-protein ACE2 inhibitory variants are expected to confer a high degree of resistance to SARS-CoV-2 infection whilst the S-protein ACE2 affinity enhancing variant may lead to additional susceptibility and severity. We also performed in silico saturation mutagenesis of the S-protein ACE2 interface and identified a further 38 potential missense mutations that could strongly inhibit binding and one more that is likely to enhance binding (Thr27Arg). A conservative estimate places the prevalence of the strongly protective variants between 12-70 per 100,000 population but there is the possibility of higher prevalence in local populations or those underrepresented in gnomAD. The probable interplay between these ACE2 affinity variants and ACE2 expression polymorphisms is highlighted as well as gender differences in penetrance arising from ACE2’s situation on the X-chromosome. It is also described how our data can help power future genetic association studies of COVID-19 phenotypes and how the saturation mutant predictions can help design a mutant ACE2 with tailored S-protein affinity, which may be an improvement over a current recombinant ACE2 that is undergoing clinical trial.Key results 1 ACE2 gnomAD missense variant (p.Gly326Glu) and one unobserved missense mutation (Thr27Arg) are predicted to enhance ACE2 binding with SARS-CoV-2 Spike protein, which could result in increased susceptibility and severity of COVID-193 ACE2 missense variants in gnomAD plus another 38 unobserved missense mutations are predicted to inhibit Spike binding, these are expected to confer a high degree of resistance to infectionThe prevalence of the strongly protective variants is estimated between 12-70 per 100,000 population but higher prevalence may exist in local populations or those underrepresented in gnomADA strategy to design a recombinant ACE2 with tailored affinity towards Spike and its potential therapeutic value is presentedThe predictions were extensively validated against published ACE2 mutant binding assays for SARS-CoV Spike protein</p> </div> </div> </div> </li> <li> <div class="row"> <div id="MacGowan2020-bd" class="col-sm-10"> <div class="title">The Dundee Resource for Sequence Analysis and Structure Prediction</div> <div class="author"> Stuart A MacGowan , <a href="https://www.ebi.ac.uk/people/person/fabio-madeira/" rel="external nofollow noopener" target="_blank">Fábio Madeira</a>, <a href="https://github.com/tbrittoborges/tbrittoborges" rel="external nofollow noopener" target="_blank">Thiago Britto-Borges</a>, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Mateusz Warowny, Alexey Drozdetskiy, James B Procter, Geoffrey J Barton' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em>Protein Sci.</em>, Jan 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="http://dx.doi.org/10.1002/pro.3783" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">URL</a> <a href="https://doi.org/10.1002/pro.3783" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>The Dundee Resource for Sequence Analysis and Structure Prediction (DRSASP; http://www.compbio.dundee.ac.uk/drsasp.html) is a collection of web services provided by the Barton Group at the University of Dundee. DRSASP’s flagship services are the JPred4 webserver for secondary structure and solvent accessibility prediction and the JABAWS 2.2 webserver for multiple sequence alignment, disorder prediction, amino acid conservation calculations, and specificity-determining site prediction. DRSASP resources are available through conventional web interfaces and APIs but are also integrated into the Jalview sequence analysis workbench, which enables the composition of multitool interactive workflows. Other existing Barton Group tools are being brought under the banner of DRSASP, including NoD (Nucleolar localization sequence detector) and 14-3-3-Pred. New resources are being developed that enable the analysis of population genetic data in evolutionary and 3D structural contexts. Existing resources are actively developed to exploit new technologies and maintain parity with evolving web standards. DRSASP provides substantial computational resources for public use, and since 2016 DRSASP services have completed over 1.5 million jobs.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Llabres2020-ot" class="col-sm-10"> <div class="title">Disease related single point mutations alter the global dynamics of a tetratricopeptide (TPR) α-solenoid domain</div> <div class="author"> Salomé Llabrés , Maxim I Tsenkov , Stuart A MacGowan , and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Geoffrey J Barton, Ulrich Zachariae' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>J. Struct. Biol.</em>, Jan 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://linkinghub.elsevier.com/retrieve/pii/S1047847719302266" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">URL</a> <a href="https://doi.org/10.1016/j.jsb.2019.107405" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Tetratricopeptide repeat (TPR) proteins belong to the class of α-solenoid proteins, in which repetitive units of α-helical hairpin motifs stack to form superhelical, often highly flexible structures. TPR domains occur in a wide variety of proteins, and perform key functional roles including protein folding, protein trafficking, cell cycle control and post-translational modification. Here, we look at the TPR domain of the enzyme O-linked GlcNAc-transferase (OGT), which catalyses O-GlcNAcylation of a broad range of substrate proteins. A number of single-point mutations in the TPR domain of human OGT have been associated with the disease Intellectual Disability (ID). By extended steered and equilibrium atomistic simulations, we show that the OGT-TPR domain acts as an elastic nanospring, and that each of the ID-related local mutations substantially affect the global dynamics of the TPR domain. Since the nanospring character of the OGT-TPR domain is key to its function in binding and releasing OGT substrates, these changes of its biomechanics likely lead to defective substrate interaction. We find that neutral mutations in the human population, selected by analysis of the gnomAD database, do not incur these changes. Our findings may not only help to explain the ID phenotype of the mutants, but also aid the design of TPR proteins with tailored biomechanical properties.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="PDBe-KB_consortium2020-wx" class="col-sm-10"> <div class="title">PDBe-KB: a community-driven resource for structural and functional annotations</div> <div class="author"> PDBe-KB consortium </div> <div class="periodical"> <em>Nucleic Acids Res.</em>, Jan 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="http://dx.doi.org/10.1093/nar/gkz853" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">URL</a> <a href="https://doi.org/10.1093/nar/gkz853" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>The Protein Data Bank in Europe-Knowledge Base (PDBe-KB, https://pdbe-kb.org) is a community-driven, collaborative resource for literature-derived, manually curated and computationally predicted structural and functional annotations of macromolecular structure data, contained in the Protein Data Bank (PDB). The goal of PDBe-KB is two-fold: (i) to increase the visibility and reduce the fragmentation of annotations contributed by specialist data resources, and to make these data more findable, accessible, interoperable and reusable (FAIR) and (ii) to place macromolecular structure data in their biological context, thus facilitating their use by the broader scientific community in fundamental and applied research. Here, we describe the guidelines of this collaborative effort, the current status of contributed data, and the PDBe-KB infrastructure, which includes the data exchange format, the deposition system for added value annotations, the distributable database containing the assembled data, and programmatic access endpoints. We also describe a series of novel web-pages-the PDBe-KB aggregated views of structure data-which combine information on macromolecular structures from many PDB entries. We have recently released the first set of pages in this series, which provide an overview of available structural and functional information for a protein of interest, referenced by a UniProtKB accession.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2018</h2> <ol class="bibliography"><li> <div class="row"> <div id="Portnoi2018-km" class="col-sm-10"> <div class="title">Mutations involving the SRY-related gene SOX8 are associated with a spectrum of human reproductive anomalies</div> <div class="author"> Marie-France Portnoi , Marie-Charlotte Dumargne , Sandra Rojo , and <span class="more-authors" title="click to view 35 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '35 more authors' ? 'Selma F Witchel, Andrew J Duncan, Caroline Eozenou, Joelle Bignon-Topalovic, Svetlana A Yatsenko, Aleksandar Rajkovic, Miguel Reyes-Mugica, Kristian Almstrup, Leila Fusee, Yogesh Srivastava, Sandra Chantot-Bastaraud, Capucine Hyon, Christine Louis-Sylvestre, Pierre Validire, Caroline Malleray Pichard, Celia Ravel, Sophie Christin-Maitre, Raja Brauner, Raffaella Rossetti, Luca Persani, Eduardo H Charreau, Liliana Dain, Violeta A Chiauzzi, Inas Mazen, Hassan Rouba, Caroline Schluth-Bolard, Stuart MacGowan, W H Irwin McLean, Etienne Patin, Ewa Rajpert-De Meyts, Ralf Jauch, John C Achermann, Jean-Pierre Siffroi, Ken McElreavey, Anu Bashamboo' : '35 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">35 more authors</span> </div> <div class="periodical"> <em>Hum. Mol. Genet.</em>, Apr 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="http://dx.doi.org/10.1093/hmg/ddy037" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">URL</a> <a href="https://doi.org/10.1093/hmg/ddy037" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>SOX8 is an HMG-box transcription factor closely related to SRY and SOX9. Deletion of the gene encoding Sox8 in mice causes reproductive dysfunction but the role of SOX8 in humans is unknown. Here, we show that SOX8 is expressed in the somatic cells of the early developing gonad in the human and influences human sex determination. We identified two individuals with 46, XY disorders/differences in sex development (DSD) and chromosomal rearrangements encompassing the SOX8 locus and a third individual with 46, XY DSD and a missense mutation in the HMG-box of SOX8. In vitro functional assays indicate that this mutation alters the biological activity of the protein. As an emerging body of evidence suggests that DSDs and infertility can have common etiologies, we also analysed SOX8 in a cohort of infertile men (n = 274) and two independent cohorts of women with primary ovarian insufficiency (POI; n = 153 and n = 104). SOX8 mutations were found at increased frequency in oligozoospermic men (3.5%; P &lt; 0.05) and POI (5.06%; P = 4.5 \times 10-5) as compared with fertile/normospermic control populations (0.74%). The mutant proteins identified altered SOX8 biological activity as compared with the wild-type protein. These data demonstrate that SOX8 plays an important role in human reproduction and SOX8 mutations contribute to a spectrum of phenotypes including 46, XY DSD, male infertility and 46, XX POI.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2017</h2> <ol class="bibliography"><li> <div class="row"> <div id="MacGowan2017-ka" class="col-sm-10"> <div class="title">Human Missense Variation is Constrained by Domain Structure and Highlights Functional and Pathogenic Residues</div> <div class="author"> Stuart A MacGowan , <a href="https://www.ebi.ac.uk/people/person/fabio-madeira/" rel="external nofollow noopener" target="_blank">Fábio Madeira</a>, <a href="https://github.com/tbrittoborges/tbrittoborges" rel="external nofollow noopener" target="_blank">Thiago Britto-Borges</a>, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Melanie S Schmittner, Christian Cole, Geoffrey J Barton' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> Apr 2017 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.biorxiv.org/content/10.1101/127050" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">URL</a> <a href="https://doi.org/10.1101/127050" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Human genome sequencing has generated population variant datasets containing millions of variants from hundreds of thousands of individuals[1][1]-[3][2]. The datasets show the genomic distribution of genetic variation to be influenced on genic and sub-genic scales by gene essentiality,[1][1],[4][3],[5][4] protein domain architecture[6][5] and the presence of genomic features such as splice donor/acceptor sites.[2][6] However, the variant data are still too sparse to provide a comparative picture of genetic variation between individual protein residues in the proteome.[1][1],[6][5] Here, we overcome this sparsity for ∼25,000 human protein domains in 1,291 domain families by aggregating variants over equivalent positions (columns) in multiple sequence alignments of sequence-similar (paralagous) domains[7][7],[8][8]. We then compare the resulting variation profiles from the human population to residue conservation across all species[9][9] and find that the same tertiary structural and functional pressures that affect amino acid conservation during domain evolution constrain missense variant distributions. Thus, depletion of missense variants at a position implies that it is structurally or functionally important. We find such positions are enriched in known disease-associated variants (OR = 2.83, p ≈0) while positions that are both missense depleted and evolutionary conserved are further enriched in disease-associated variants (OR = 1.85, p = 3.3\times10-17) compared to those that are only evolutionary conserved (OR = 1.29, p = 4.5\times10-19). Unexpectedly, a subset of evolutionary Unconserved positions are Missense Depleted in human (UMD positions) and these are also enriched in pathogenic variants (OR = 1.74, p = 0.02). UMD positions are further differentiated from other unconserved residues in that they are enriched in ligand, DNA and protein binding interactions (OR = 1.59, p = 0.003), which suggests this stratification can identify functionally important positions. A different class of positions that are Conserved and Missense Enriched (CME) show an enrichment of ClinVar risk factor variants (OR = 2.27, p = 0.004). We illustrate these principles with the G-Protein Coupled Receptor (GPCR) family, Nuclear Receptor Ligand Binding Domain family and In Between Ring-Finger (IBR) domains and list a total of 343 UMD positions in 211 domain families. This study will have broad applications to: (a) providing focus for functional studies of specific proteins by mutagenesis; (b) refining pathogenicity prediction models; (c) highlighting which residue interactions to target when refining the specificity of small-molecule drugs. [1]: #ref-1 [2]: #ref-3 [3]: #ref-4 [4]: #ref-5 [5]: #ref-6 [6]: #ref-2 [7]: #ref-7 [8]: #ref-8 [9]: #ref-9</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2016</h2> <ol class="bibliography"><li> <div class="row"> <div id="MacGowan2016-fv" class="col-sm-10"> <div class="title">Contribution of bacteriochlorophyll conformation to the distribution of site-energies in the FMO protein</div> <div class="author"> Stuart A MacGowan , and Mathias O Senge </div> <div class="periodical"> <em>Biochim. Biophys. Acta</em>, Apr 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="http://dx.doi.org/10.1016/j.bbabio.2016.02.001" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">URL</a> <a href="https://doi.org/10.1016/j.bbabio.2016.02.001" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>The structural data for the Fenna-Matthews-Olson (FMO) protein indicate that the bacteriochlorophylls (BChls) display a significant degree of conformational heterogeneity of their peripheral substituents and the protein-induced nonplanar skeletal deformations of the tetrapyrrole macrocycle. As electronic properties of chromophores are altered by such differences, a conformational effect may influence the site-energies of specific pigments and thus play a role in mediating the excitation energy transfer dynamics, but this has not yet been established. The difficulty of assessing this question is shown to be partly the result of the inability of the sequential truncation approach usually employed to account for interactions between the conformations of the macrocycle and its substituents and an alternative approach is suggested. By assigning the BChl atoms to meaningful atom groups and performing all possible permutations of partial optimizations in a full-factorial design, where each group is either frozen in the crystal geometry or optimized in vacuo, followed by excited state calculations on each resulting structure (PM6//ZIndo/S), the specific effects of the conformations of each BChl component as well as mutual interactions between the molecular fragments on the site-energy can be delineated. This factorial relaxation procedure gives different estimates of the macrocycle conformational perturbation than the approach of sequentially truncating the BChl periphery. The results were evaluated in the context of published site-energies for the FMO pigments from three species to identify how conformational effects contribute to their distribution and instances of cross-species conservation and functional divergence of the BChl nonplanarity conformational contribution are described.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2015</h2> <ol class="bibliography"> <li> <div class="row"> <div id="Mercier2015-of" class="col-sm-10"> <div class="title">Expanding the clinical spectrum of hereditary fibrosing poikiloderma with tendon contractures, myopathy and pulmonary fibrosis due to FAM111B mutations</div> <div class="author"> Sandra Mercier , Sébastien Küry , Emmanuelle Salort-Campana , and <span class="more-authors" title="click to view 51 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '51 more authors' ? 'Armelle Magot, Uchenna Agbim, Thomas Besnard, Nathalie Bodak, Chantal Bou-Hanna, Flora Bréhéret, Perrine Brunelle, Florence Caillon, Brigitte Chabrol, Valérie Cormier-Daire, Albert David, Bruno Eymard, Laurence Faivre, Dominique Figarella-Branger, Emmanuelle Fleurence, Mythily Ganapathi, Romain Gherardi, Alice Goldenberg, Antoine Hamel, Jeanine Igual, Alan D Irvine, Dominique Israël-Biet, Caroline Kannengiesser, Christian Laboisse, Cédric Le Caignec, Jean-Yves Mahé, Stéphanie Mallet, Stuart MacGowan, Maeve A McAleer, Irwin McLean, Cécile Méni, Arnold Munnich, Jean-Marie Mussini, Peter L Nagy, Jeffrey Odel, Grainne M O’Regan, Yann Péréon, Julie Perrier, Juliette Piard, Eve Puzenat, Jacinda B Sampson, Frances Smith, Nadem Soufir, Kurenai Tanji, Christel Thauvin, Christina Ulane, Rosemarie M Watson, Nonhlanhla P Khumalo, Bongani M Mayosi, Sébastien Barbarot, Stéphane Bézieau' : '51 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">51 more authors</span> </div> <div class="periodical"> <em>Orphanet J. Rare Dis.</em>, Oct 2015 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="http://dx.doi.org/10.1186/s13023-015-0352-4" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">URL</a> <a href="https://doi.org/10.1186/s13023-015-0352-4" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>BACKGROUND: Hereditary Fibrosing Poikiloderma (HFP) with tendon contractures, myopathy and pulmonary fibrosis (POIKTMP [MIM 615704]) is a very recently described entity of syndromic inherited poikiloderma. Previously by using whole exome sequencing in five families, we identified the causative gene, FAM111B (NM_198947.3), the function of which is still unknown. Our objective in this study was to better define the specific features of POIKTMP through a larger series of patients. METHODS: Clinical and molecular data of two families and eight independent sporadic cases, including six new cases, were collected. RESULTS: Key features consist of: (i) early-onset poikiloderma, hypotrichosis and hypohidrosis; (ii) multiple contractures, in particular triceps surae muscle contractures; (iii) diffuse progressive muscular weakness; (iv) pulmonary fibrosis in adulthood and (v) other features including exocrine pancreatic insufficiency, liver impairment and growth retardation. Muscle magnetic resonance imaging was informative and showed muscle atrophy and fatty infiltration. Histological examination of skeletal muscle revealed extensive fibroadipose tissue infiltration. Microscopy of the skin showed a scleroderma-like aspect with fibrosis and alterations of the elastic network. FAM111B gene analysis identified five different missense variants (two recurrent mutations were found respectively in three and four independent families). All the mutations were predicted to localize in the trypsin-like cysteine/serine peptidase domain of the protein. We suggest gain-of-function or dominant-negative mutations resulting in FAM111B enzymatic activity changes. CONCLUSIONS: HFP with tendon contractures, myopathy and pulmonary fibrosis, is a multisystemic disorder due to autosomal dominant FAM111B mutations. Future functional studies will help in understanding the specific pathological process of this fibrosing disorder.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Senge2015-bb" class="col-sm-10"> <div class="title">Conformational control of cofactors in nature - the influence of protein-induced macrocycle distortion on the biological function of tetrapyrroles</div> <div class="author"> Mathias O Senge , Stuart A MacGowan , and Jessica M O’Brien </div> <div class="periodical"> <em>Chem. Commun.</em>, Dec 2015 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="http://dx.doi.org/10.1039/c5cc06254c" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">URL</a> <a href="https://doi.org/10.1039/c5cc06254c" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Tetrapyrrole-containing proteins are one of the most fundamental classes of enzymes in nature and it remains an open question to give a chemical rationale for the multitude of biological reactions that can be catalyzed by these pigment-protein complexes. There are many fundamental processes where the same (i.e., chemically identical) porphyrin cofactor is involved in chemically quite distinct reactions. For example, heme is the active cofactor for oxygen transport and storage (hemoglobin, myoglobin) and for the incorporation of molecular oxygen in organic substrates (cytochrome P450). It is involved in the terminal oxidation (cytochrome c oxidase) and the metabolism of H2O2 (catalases and peroxidases) and catalyzes various electron transfer reactions in cytochromes. Likewise, in photosynthesis the same chlorophyll cofactor may function as a reaction center pigment (charge separation) or as an accessory pigment (exciton transfer) in light harvesting complexes (e.g., chlorophyll a). Whilst differences in the apoprotein sequences alone cannot explain the often drastic differences in physicochemical properties encountered for the same cofactor in diverse protein complexes, a critical factor for all biological functions must be the close structural interplay between bound cofactors and the respective apoprotein in addition to factors such as hydrogen bonding or electronic effects. Here, we explore how nature can use the same chemical molecule as a cofactor for chemically distinct reactions using the concept of conformational flexibility of tetrapyrroles. The multifaceted roles of tetrapyrroles are discussed in the context of the current knowledge on distorted porphyrins. Contemporary analytical methods now allow a more quantitative look at cofactors in protein complexes and the development of the field is illustrated by case studies on hemeproteins and photosynthetic complexes. Specific tetrapyrrole conformations are now used to prepare bioengineered designer proteins with specific catalytic or photochemical properties.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="McAleer2015-od" class="col-sm-10"> <div class="title">Severe dermatitis, multiple allergies, and metabolic wasting syndrome caused by a novel mutation in the N-terminal plakin domain of desmoplakin</div> <div class="author"> Maeve A McAleer , Elizabeth Pohler , Frances J D Smith , and <span class="more-authors" title="click to view 17 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '17 more authors' ? 'Neil J Wilson, Christian Cole, Stuart MacGowan, Jennifer L Koetsier, Lisa M Godsel, Robert M Harmon, Robert Gruber, Debra Crumrine, Peter M Elias, Michael McDermott, Karina Butler, Annemarie Broderick, Ofer Sarig, Eli Sprecher, Kathleen J Green, W H Irwin McLean, Alan D Irvine' : '17 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">17 more authors</span> </div> <div class="periodical"> <em>J. Allergy Clin. Immunol.</em>, Nov 2015 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="http://dx.doi.org/10.1016/j.jaci.2015.05.002" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">URL</a> <a href="https://doi.org/10.1016/j.jaci.2015.05.002" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>BACKGROUND: Severe dermatitis, multiple allergies, and metabolic wasting (SAM) syndrome is a recently recognized syndrome caused by mutations in the desmoglein 1 gene (DSG1). To date, only 3 families have been reported. OBJECTIVE: We studied a new case of SAM syndrome known to have no mutations in DSG1 to detail the clinical, histopathologic, immunofluorescent, and ultrastructural phenotype and to identify the underlying molecular mechanisms in this rare genodermatosis. METHODS: Histopathologic, electron microscopy, and immunofluorescent studies were performed. Whole-exome sequencing data were interrogated for mutations in desmosomal and other skin structural genes, followed by Sanger sequencing of candidate genes in the patient and his parents. RESULTS: No mutations were identified in DSG1; however, a novel de novo heterozygous missense c.1757A&gt;C mutation in the desmoplakin gene (DSP) was identified in the patient, predicting the amino acid substitution p.His586Pro in the desmoplakin polypeptide. CONCLUSIONS: SAM syndrome can be caused by mutations in both DSG1 and DSP. Knowledge of this genetic heterogeneity is important for both analysis of patients and genetic counseling of families. This condition and these observations reinforce the importance of heritable skin barrier defects, in this case desmosomal proteins, in the pathogenesis of atopic disease.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2014</h2> <ol class="bibliography"> <li> <div class="row"> <div id="MacGowan2014-dy" class="col-sm-10"> <div class="title">Conformational Control of Cofactors in Nature: The Influence of Protein Induced Macrocycle Distortion on the Biological Function of Tetrapyrroles in Photosynthesis</div> <div class="author"> Stuart A MacGowan </div> <div class="periodical"> <em>Trinity College Dublin</em> , Nov 2014 </div> <div class="periodical"> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div id="Senge2014-qn" class="col-sm-10"> <div class="title">Chlorophylls, Symmetry, Chirality, and Photosynthesis</div> <div class="author"> Mathias O Senge , Aoife A Ryan , Kristie A Letchford , and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Stuart A MacGowan, Tamara Mielke' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>Symmetry</em>, Sep 2014 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.mdpi.com/2073-8994/6/3/781" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">URL</a> <a href="https://doi.org/10.3390/sym6030781" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Chlorophylls are a fundamental class of tetrapyrroles and function as the central reaction center, accessory and photoprotective pigments in photosynthesis. Their unique individual photochemical properties are a consequence of the tetrapyrrole macrocycle, the structural chemistry and coordination behavior of the phytochlorin system, and specific substituent pattern. They achieve their full potential in solar energy conversion by working in concert in highly complex, supramolecular structures such as the reaction centers and light-harvesting complexes of photobiology. The biochemical function of these structures depends on the controlled interplay of structural and functional principles of the apoprotein and pigment cofactors. Chlorophylls and bacteriochlorophylls are optically active molecules with several chiral centers, which are necessary for their natural biological function and the assembly of their supramolecular complexes. However, in many cases the exact role of chromophore stereochemistry in the biological context is unknown. This review gives an overview of chlorophyll research in terms of basic function, biosynthesis and their functional and structural role in photosynthesis. It highlights aspects of chirality and symmetry of chlorophylls to elicit further interest in their role in nature.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2013</h2> <ol class="bibliography"><li> <div class="row"> <div id="MacGowan2013-wl" class="col-sm-10"> <div class="title">Computational quantification of the physicochemical effects of heme distortion: redox control in the reaction center cytochrome subunit of Blastochloris viridis</div> <div class="author"> Stuart A MacGowan , and Mathias O Senge </div> <div class="periodical"> <em>Inorg. Chem.</em>, Feb 2013 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="http://dx.doi.org/10.1021/ic301530t" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">URL</a> <a href="https://doi.org/10.1021/ic301530t" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>A facile, experimentally calibrated computational procedure is described that affords the relative ordering of heme cofactor reduction potentials with respect to intrinsic shifts brought about by apoprotein induced heme-macrocycle distortion. The method utilizes heme-Fe partial atomic charges and is useful with the computationally inexpensive B3LYP/3-21g method calculated for simplified heme models extracted from the Protein Data Bank incorporating only the effects of varying macrocycle conformations and thereby delineating their physicochemical effects. The procedure was successfully calibrated using the atomic coordinates and published midpoint potentials from the heme cofactors in wild-type and a series of heme-NO and -O(2) binding domain mutants and thus confirmed the sole conformational modulation of the redox potentials in these complexes. This technique was also applied to the reaction center tetraheme cytochrome subunit of Blastochloris viridis to build upon previous work elucidating the role that conformational control plays in photosynthetic systems, and it was found that this effect may account for up to 70% (54mv) of the observed differences in the reduction potentials of the four hemes. We validate the approach using larger basis sets up to and including the triple-ζ, doubly polarized and augmented 6-311+g** basis and discuss the specific conformational origins of the effect.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2011</h2> <ol class="bibliography"><li> <div class="row"> <div id="MacGowan2011-pf" class="col-sm-10"> <div class="title">Conformational control of cofactors in nature–functional tetrapyrrole conformations in the photosynthetic reaction centers of purple bacteria</div> <div class="author"> Stuart A MacGowan , and Mathias O Senge </div> <div class="periodical"> <em>Chem. Commun.</em>, Nov 2011 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="http://dx.doi.org/10.1039/c1cc14686f" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">URL</a> <a href="https://doi.org/10.1039/c1cc14686f" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Chemically identical tetrapyrrole cofactors such as hemes and chlorophylls participate in functionally diverse biological roles. An analysis of the available protein structural data for the bacteriochlorophylls in the photosynthetic reaction center gives statistically reliable evidence of the hypothesis that the protein induced cofactor conformation is a modulator of the bio-molecular function of each reaction center. The results serve as a general model to illustrate conformational control of tetrapyrrole cofactors in other proteins.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2010</h2> <ol class="bibliography"><li> <div class="row"> <div id="Senge2010-sl" class="col-sm-10"> <div class="title">The structural chemistry of isolated chlorophylls</div> <div class="author"> Mathias O Senge , and Stuart A MacGowan </div> <div class="periodical"> <em>Handbook of porphyrin science</em>, Nov 2010 </div> <div class="periodical"> </div> <div class="links"> </div> </div> </div> </li></ol> </div> </article> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> © Copyright 2024 Stuart A. MacGowan. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. Photos from <a href="https://unsplash.com" target="_blank" rel="external nofollow noopener">Unsplash</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?5d75c11f89cd96294bf5e6dd1ee1bb30"></script> <script defer src="/assets/js/common.js?fcfacfb8c6281f5e68d5a7d348186eb1"></script> <script defer src="/assets/js/copy_code.js?12775fdf7f95e901d7119054556e495f" type="text/javascript"></script> <script defer src="/assets/js/jupyter_new_tab.js?d9f17b6adc2311cbabd747f4538bb15f"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6"></script> <script async src="https://www.googletagmanager.com/gtag/js?id=G-ZNT0W2MVX3"></script> <script>function gtag(){window.dataLayer.push(arguments)}window.dataLayer=window.dataLayer||[],gtag("js",new Date),gtag("config","G-ZNT0W2MVX3");</script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> </body> </html>